main.go

```diff
@@ -192,7 +192,16 @@ func (c *nodeLocalController) SetupReconcilers(mgr manager.Manager, opts *nnf.Op
 		return err
 	}
 
-	if os.Getenv("ENVIRONMENT") != "kind" {
```

Comment:
If I'm reading this right, you can just pass the if condition to the FakeMount param
```go
		if err := (&controllers.NnfClientMountReconciler{
			Client:    mgr.GetClient(),
			Log:       ctrl.Log.WithName("controllers").WithName("NnfClientMount"),
			Scheme:    mgr.GetScheme(),
			FakeMount: os.Getenv("ENVIRONMENT") == "kind",
		}).SetupWithManager(mgr); err != nil {
			return err
		}
```

Reply:
Or, there appears to be only one spot where FakeMount is used, and it could be replaced by the conditional `if os.Getenv("ENVIRONMENT") == "kind" {` like you did in the nnf_access_controller. I prefer this instead of expanding the struct since searching for `== "kind"` would be consistent.

Reply:
Sounds good. I'll change it